### ISSUE TITLE
Modifying parent properties triggers repaint

### ISSUE THREAD

**Issue by danieltamas** [alexjoverm/v-runtime-template#56]
When the parent's data is modified from the template, it triggers
a repaint on the parent.

This breaks already initialised components ( like sliders )

**Comment by pascalporedda** [comment]
Yep experiencing the same behaviour, wondering if that is somewhat fixable?

**Comment by neberaa** [comment]
Faced with the same problem creating tabs. Each tab may contains html raw with uncompiled Vue js component cames from backend. `v-runtime-template` create/destroy each component on every tab change ((

**Comment by danieltamas** [comment]
The problem I suspect is the parent node gets merged into the new node, whereas it should be the other way around, to avoid destroying the parent node

**Comment by neberaa** [comment]
In my case i'm just switching a css class. Nothing else. Anyway it should not be rerendered each time on class list change. Definitely `keep-alive` property need to be implemented (as native dynamic components have)

**Comment by SikoSoft** [comment]
I am experiencing infinite loop event handling, which is the result of this bug.

In an attempt to take better control over when a component's images are all loaded, I tried this library, then attached @load to the images, which in turn triggers an event handler that increments the number of images loaded.

Because the handler increments the component's "imagesLoaded" property, a re-render is performed, causing an infinite loop.

This is a major performance glitch which renders this library unfortunately unusable for me.

Since this project is no longer maintained, I'd be very careful when using it because it has great potential for harm as it is now!
